### PR TITLE
feat(stock): add expiration date to stock exit

### LINKED ISSUE #5375: Add the expiration date in dropdown of lots in the stock exit module
This issue is about to add the lot's expiration date next to the lot name in the dropdown of selecting lot

image

### PR DESCRIPTION
**Pull request by jniles**
Adds the expiration date to the stock exit lot selection.

Closes #5375.

image

### CODE REVIEW

**Review by jmcameron — COMMENTED** [review]
The code looks good.
I tested this with the IMCK db and it works fine.
However, since there is room, I suggest adding "Expires" next to the date so the display for the lots would look like:
  Lot-ABC (expires 23/8/2022)

That way noone has to guess what the date refers to.

**Comment by jniles** [comment]
Done:

image

File: client/src/modules/stock/exit/templates/lot.tmpl.html [hunk 1]
```diff
@@ -7,9 +7,8 @@
     <ui-select-match placeholder="{{ ::'FORM.PLACEHOLDERS.LOT' | translate }}">
       <span>{{$select.selected.label}}</span>
     </ui-select-match>
-    <ui-select-choices
-      repeat="lot as lot in row.entity.lots | filter: $select.search">
-      <span>{{ lot.label }} </span>
+    <ui-select-choices repeat="lot as lot in row.entity.lots | filter: $select.search">
+      <span title="{{lot.expiration_date | date:grid.appScope.DATE_FMT}}">{{ lot.label }} ({{ "LOTS_SCHEDULE.LOT_EXPIRES" | translate }} {{lot.expiration_date | date:grid.appScope.DATE_FMT}})</span>
```
Comment (jmcameron) [inline client/src/modules/stock/exit/templates/lot.tmpl.html:11]: Nice!

**Review by jmcameron — APPROVED** [review]
Looks good!
I tested it and it seems very clear

**Pull request merged**